src/App.js

```diff
@@ -1,84 +1,99 @@
 import React, { Component } from 'react';
 import './App.css';
-
-import Home from './component/Home';
-import Completed from './component/Completed';
-import Remaining from './component/Remaining';
+import Container from './component/Container';
+import AddTodoInput from './component/AddTodoInput';
+import Header from './component/Header';
+import Tabs from './component/Tabs';
```

Comment:
Let's reorder the imports in pyramidical order so that it is easier to read.

```
import Tabs from './component/Tabs';
import Header from './component/Header';
import Container from './component/Container';
import AddTodoInput from './component/AddTodoInput';
```